### ISSUE TITLE
Blood Fury

### ISSUE THREAD

**Issue by zytexa** [pwatts6060/SuppliesTrackerExternal#91]
I see its not being track, would it be a possible addition?

**Comment by pwatts6060** [comment]
Yes, likely will add this when/if they give a 1 minimum hit as currently you can't differentiate between hitting a 0 which would use a charge and missing which also shows a 0 but doesn't use a charge and I'm not wanting to calculate maxhit and use probability as it's too much work. It will be much easier to add correctly in the future.

**Comment by pwatts6060** [comment]
This can be added now with the successful accuracy rolls always showing a distinguishable 1+ damage hit now. That said it can be done either via xp drops (check for abscence of same tick range/mage xp on defensive, 200m xp use fake xp drops, cannon xp drop + defensive xp drop same tick consideration) or melee attack anims (huge list of possible animations), both of which aren't very simple to do. 

If someone wants to tackle this feel free

**Comment by Im2be** [comment]
The Weapon Charges plugin already does something like this. It's not always 100% accurate (e.g. when thrall hits coincide with a scythe multi-hitsplat). However, it should still work well enough for the supplies tracker:
https://github.com/search?q=repo%3Ageheur%2Fweapon-charges%20blood%20fury&type=code
